File: java-solutions/src/main/java/org/karabalin/s2678/Solution2678.java
package org.karabalin.s2678;

import org.junit.jupiter.api.Assertions;

class Solution {
    public int countSeniors(String[] details) {
        int result = 0;
        for (String s : details) {
            if (Integer.parseInt(s.substring(11, 13)) > 60) {
                ++result;
            }
        }
        return result;
    }
}

public class Solution2678 {
    public static void main(String[] args) {
        var s = new Solution();
        Assertions.assertEquals(2, s.countSeniors(new String[]{"7868190130M7522", "5303914400F9211", "9273338290F4010"}));
    }
}

File: kotlin/src/main/kotlin/org/karabalin/f2001t3000/f2601t2700/s2678/Solution2678.kt
package org.karabalin.f2001t3000.f2601t2700.s2678

class Solution {
    fun countSeniors(details: Array<String>): Int {
        var result = 0
        for (s in details) {
            val age = s.substring(11, 13).toInt()
            if (age > 60) {
                result++
            }
        }
        return result
    }
}

fun main() {
    val solution = Solution()
    val details = arrayOf(
        "7868190130M7522",
        "5303914400F9211",
        "9273338290F4010"
    )
    println(solution.countSeniors(details))
}
